src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj

```diff
@@ -43,26 +51,13 @@
   <PropertyGroup Label="Configuration">
     <VCToolsVersion />
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32' OR '$(Configuration)|$(Platform)'=='Debug|x64' OR '$(Configuration)|$(Platform)'=='Debug|ARM64' OR '$(Configuration)|$(Platform)'=='Debug|Any CPU'" Label="Configuration">
```

Comment:
What platforms are you rejecting here (i.e. could you just check `Configuration` = `Debug`?)

Reply:
Do we have a regular ARM (not 64) bit platform that we build still?  I believe that's the only one that's still split out from the main windows job on CI too